com.Genymobile.scrcpy.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "com.Genymobile.scrcpy",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "scrcpy",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "19.08",
+            "add-ld-path": "."
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "modules": [
+        {
+            "name": "scrcpy",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D scrcpy-server-v1.13 /app/share/scrcpy-server.jar",
+                "meson x --buildtype release --strip -Db_lto=true -Dprebuilt_server=/app/share/scrcpy-server.jar",
+                "ninja -C x",
+                "install -D x/app/scrcpy /app/bin/scrcpy",
+		"install -D com.Genymobile.scrcpy.metainfo.xml -t /app/share/metainfo"
```

Comment:
You forgot to include the com.Genymobile.scrcpy.metainfo.xml file in sources.

Reply:
By the way, I am not sure why are you including both the git and file scrcpy sources.

Reply:
thanks,it is mistake

Reply:
Thanks. The first repo is used to build client. The second file is a prebuild program need to be pushed to other device.